content/en/docs/reference/kubectl/quick-reference.md

```diff
@@ -71,12 +71,19 @@ KUBECONFIG=~/.kube/config:~/.kube/kubconfig2
 
 kubectl config view
 
+# Show merged kubeconfig settings and raw certificate data and exposed secrets
+kubectl config view --raw 
+
 # get the password for the e2e user
 kubectl config view -o jsonpath='{.users[?(@.name == "e2e")].user.password}'
 
+# get the certificate for the e2e user
+kubectl config view --raw -ojsonpath="{.users[?(.name == 'e2e')].user.client-certificate-data}" | base64 -d
```

Comment:
```suggestion
kubectl config view --raw -ojsonpath='{.users[?(.name == 'e2e')].user.client-certificate-data}' | base64 -d
```

Reply:
single quote worked for me but not the double quote version. is the double quote correct way?

Reply:
it works as well